app/vue/contexts/profile/ProfileDetailsPageContext.js

```diff
@@ -263,6 +282,29 @@ export default class ProfileDetailsContext extends BaseFuroContext {
     }
   }
 
+  /**
+   * get: competitionParticipantCapsule
+   *
+   * @returns {import('~/app/graphql/client/queries/competitionParticipant/CompetitionParticipantQueryGraphqlCapsule').default | null}
+   */
+  get competitionParticipantCapsule () {
+    return this.graphqlClientHash
+      .competitionParticipant
+      .capsuleRef
+      .value
+  }
+
+  /**
+   * get: competitionParticipantStatusId
+   *
+   * @returns {number | null}
+   */
+  get competitionParticipantStatusId () {
+    return this.competitionParticipantCapsule
+      ?.statusId
+      ?? null
```

Comment:
@hecticme 

Since Capsule has the responsibility to resolve `null`, we can write this getter as follows:

```suggestion
    return this.competitionParticipantCapsule.statusId
```